audio/filter/af_lavrresample.c

```diff
@@ -400,10 +400,13 @@ static int control(struct af_instance *af, int cmd, void *arg)
     case AF_CONTROL_RESET:
         if (s->avrctx) {
 #if HAVE_LIBSWRESAMPLE
-            // This POS either can't drop state correctly, or doesn't want to.
-            // It will swallow some minor audio e.g. after a seek.
-            // Deallocate and recreate the resample state for a full reset.
-            configure_lavrr(af, &af->fmt_in, &af->fmt_out, false);
+            int r;
+
+            swr_close(s->avrctx);
+            if ((r = swr_init(s->avrctx)) < 0) {
+                close_lavrr(af);
+                return r;
```

Comment:
No, mpv doesn't use ffmpeg error codes internally. Just don't return at all. close_lavrr sets avrctx to NULL, which will take appropriate actions when filtering is attempted.